platform-emf/src/main/java/com/softicar/platform/emf/form/EmfForm.java

```diff
@@ -122,9 +124,25 @@ public void focusFrame() {
 
 	// ------------------------------ behavioral configuration ------------------------------ //
 
+	/**
+	 * Sets a callback {@link Consumer} to be called whenever a new
+	 * {@link IEmfTableRow} was inserted through this {@link EmfForm}.
+	 * <p>
+	 * This callback mechanism is primarily intended for UI updates. It is
+	 * called when the new {@link IDbTableRow} was <b>persistently</b> inserted
```

Comment:
We could add some emphasis here, like:
```suggestion
	 * called when the new {@link IDbTableRow} was already <b>persistently</b> inserted
```